addons/l10n_pe/models/account_tax.py

```diff
@@ -52,6 +58,12 @@ class AccountTaxTemplate(models.Model):
         help="Follow the UN/ECE 5305 standard from the United Nations Economic Commission for Europe for more "
              "information  http://www.unece.org/trade/untdid/d08a/tred/tred5305.htm"
     )
+    l10n_pe_edi_isc_type = fields.Selection([
+        ('01', 'System to value'),
+        ('02', 'Application of the Fixed Amount'),
+        ('03', 'Retail Price System'),
+    ], 'ISC Type',
+        help='Used in Selective Consumption Tax to indicate the type of calculation for the ISC.')
 
     def _get_tax_vals(self, company, tax_template_to_tax):
         val = super()._get_tax_vals(company, tax_template_to_tax)
```

Comment:
This method should be updated to also have this field: https://github.com/odoo/odoo/blob/230014b8af9853eef34062df431cb3cdaca9bfea/addons/l10n_pe/models/account_tax.py#L58